src/shared/components/results/Results.tsx

```diff
@@ -72,6 +77,36 @@ const Results = () => {
 
   const [params] = getParamsFromURL(search);
 
+  // Some cross references legitimately have colons in their ids eg PTHR34313:SF2
+  // The API returns "'PTHR34313' is not a valid search field" on those occassions
+  // so to assist the users, escape the colon PTHR34313\:SF2 and try a head request.
```

Comment:
You would also need to check the number of results.
Even for a search returning 0 results the API returns 200 and an empty payload, see search for "PTHR:34313SF2" for example